CMakeLists.txt

```diff
@@ -28,6 +32,8 @@ option(RADIUM_GENERATE_LIB_PLUGINBASE "Include Radium::PluginBase in CMake proje
 option(RADIUM_UPDATE_VERSION          "Update version file each time the project is compiled (update compilation time in version.cpp)" ON)
 option(RADIUM_INSTALL_DOC             "Install documentation. If RadiumDoc is compiled, install documentation to bundle directory for install target" ON)
 
+set(RADIUM_EXTERNAL_CMAKE_INSTALL_MESSAGE NEVER)
```

Comment:
When installing externals, the install messages are still there. (installing, updating, ...). I tested on Linux with cmake 3.16.3

Reply:
Yes, see my other comment about the option.

Reply:
well it seems there still is an issue here. If I run install in the external project subdir (e.g. /build/src/Core/external/OpenMesh-prefix/src/OpenMesh-build) with CMAKE_INSTALL_MESSAGE to NEVER (cmake -DCMAKE_INSTALL_MESSAGE=NEVER . && make install ), I get the expected behavior, but not if I run from the external subdir ( e.g. build/Core/external ), as there is no install target, just OpenMesh-install, don't know if it's linked.
BTW after reconfiguring the external in its build dir, there is no install messages ... maybe something went wrong during configuration.